deployment/terraform/job-definitions/default.json

```diff
@@ -0,0 +1,16 @@
+{
+  "image": "${image_url}",
+  "vcpus": ${default_vcpus},
+  "memory": ${default_memory},
+  "command": ["--version"],
```

Comment:
For the sake of simplicity I initialized the job definition to inherit from the app Gunicorn container and perform a dummy task. It seems like the ultimate goal for this job definition is to [read a row from the database and process it](https://github.com/open-apparel-registry/open-apparel-registry/issues/161), so if there's some more relevant work that we could do upfront to prove out important functionality (e.g. publish a new container image with a `python` entrypoint and use it in a job definition to ping the database via Django ORM) let me know and I can add it on.